ui/webui/src/components/HeaderKebab.jsx

```diff
@@ -135,40 +130,43 @@ export const HeaderKebab = ({ reportLinkURL, isConnected }) => {
         setIsReportIssueOpen(true);
     };
 
-    const dropdownItems = [
-        <DropdownItem id="about-modal-dropdown-item-about" key="about" onClick={handleAboutModal}>
-            {_("About")}
-        </DropdownItem>,
-        <DropdownItem id="about-modal-dropdown-item-report" key="report issue" onClick={handleReportIssue}>
-            {_("Report Issue")}
-        </DropdownItem>,
-    ];
     return (
         <>
             <Dropdown
-              position={DropdownPosition.right}
               onSelect={onSelect}
-              toggle={
-                  <KebabToggle
-                    id="toggle-kebab"
-                    onToggle={(_event, isOpen) => onToggle(isOpen)}
-                  />
-              }
               isOpen={isOpen}
-              isPlain
-              dropdownItems={dropdownItems}
-            />
+              popperProps={{ position: "right" }}
+              toggle={toggleRef =>
+                  <MenuToggle
+                    id="toggle-kebab"
+                    ref={toggleRef}
+                    variant="plain"
+                    onClick={onToggle}
+                    isExpanded={isOpen}
+                    style={{ position: "absolute", right: "0" }}>
```

Comment:
Where did you see this code being needed? I just checked the example code of https://www.patternfly.org/components/menus/dropdown and I don't see any custom style and Icon.

Reply:
Yes, but unfortunately the DropdownPosition argument has been removed and the new props for Dropdown don't seem to work as they should. Maybe it's a bug in patternfly

Reply:
But this setting actually only applies to the popper itself and does not apply to the entire dropdown. So the dropdown alignment will probably have to be solved somehow via styles

Reply:
but the current solution is unsatisfactory because it breaks the vertical alignment